packages/replay/src/replay.ts

```diff
@@ -182,6 +182,7 @@ export class ReplayContainer implements ReplayContainerInterface {
 
     this.eventBuffer = createEventBuffer({
       useCompression: this._options.useCompression,
+      recordingMode: this.recordingMode,
```

Comment:
Shouldn't, it only means that (for now) we will continue in partitioned mode for compression - meaning even after the error was sent, we'll still not be using the streaming compression but the "all at once" compression. We may adjust this in the future, but this'll mean to swap out the event buffer at runtime, which I wanted to avoid for this first step.

Maybe it is clearer to rename this, e.g. instead of `recordingMode` make it `rememberLastCheckout: boolean`? Then it's a bit clearer what it does and that it is not really _directly_ related to the recording mode?

Reply:
changed this to `keepLastCheckout: boolean`, which should hopefully be clearer!